File: modules/runtime/debugguiwindows/source/debugguiwindows.cpp

#include "tiki/debugguiwindows/debugguiwindows.hpp"

namespace tiki
{
	void DebugGuiWindows::create( DebugGui& debugGui )
	{
		m_windowDebugProp.create( debugGui );
	}

	void DebugGuiWindows::dispose()
	{
		m_windowDebugProp.dispose();
	}
}

File: engine/modules/graveyard/debugguiwindows/debugguiwindows.lua
-- library/modules/runtime/debugguiwindows

local module = Module:new( "debugguiwindows" );

module:add_files( "include/**/*.hpp" );
module:add_files( "source/*.*" );
module:add_files( "debugguiwindows.lua" );
module:add_include_dir( "include" );

module:add_dependency( "debuggui" );
